app/supplejack/api/utils/notice_harvesting_to_api.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Api
+  module Utils
+    class NoticeHarvestingToApi
```

Comment:
Just a little thing but I feel like the name of this class is a bit odd, it doesn't really read grammatically. Maybe just `NotifyHarvesting`? So the module would be `Api::Utils::NotifyHarvesting`

Reply:
I wasn't satisfied with it either. I'll change it to NotifyHarvesting, that's better